pkg/networkmanager/firewall.jsx

```diff
@@ -628,7 +628,7 @@ class AddEditServicesModal extends React.Component {
                                    id="add-services-dialog--services"
                                    value="services"
                                    isChecked={!this.state.custom}
-                                   onChange={this.onToggleType}
+                                   onChange={(_event, value) => this.onToggleType(value)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18806-20230520-070544-54b4f0d5-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)